src/rhub/api/__init__.py

```diff
@@ -62,11 +56,8 @@ def get_vault() -> Vault:
 
 def init_app():
     logger.info('Starting inititialization...')
-    for name in MODULES:
-        module = importlib.import_module(f'rhub.{name}')
-        if hasattr(module, 'init'):
-            logger.info(f'Initializing "{name}" module...')
-            module.init()
+    from ._setup import setup
```

Comment:
Is the import okay here?

Reply:
I had some problems with circular imports when it was on top of the file. `_setup` module is used only in init so I guess it is ok in this case.